### DISCUSSION TITLE
Kubernetes Horizontal Scaling

### DISCUSSION THREAD

**Discussion by NoureddinBakir** [pocketbase/pocketbase#615] [Q&A]
Since PocketBase scales vertically, would there be any theoretical drawback to scaling horizontally using Kubernetes?

**Comment by ganigeorgiev** [comment]
I don't think Kubernetes on its own could help with the horizontal scaling.

Recently Marmot was posted in https://github.com/pocketbase/pocketbase/discussions/487, which implement "multi-raft" style of scaling (aka. multiple master nodes that sync with each others), but you'll still need somehow to sync the realtime events, triggered hooks, etc. with each application.

At least until v1.0.0, I'm not planning to support horizontal scaling out of the box and if you really need it, there are plenty of other open source projects that are designed for those cases (Supabase, Nhost, etc.).
